juriscraper/opinions/united_states/state/colo.py

```diff
@@ -175,6 +177,17 @@ def _extract_cases_from_sub_page(self, html_tree, date_obj):
                 url = alt_url if alt_url else url
                 name = alt_name if alt_name else name
 
+            # In some weird cases, subpages are empty and don't include any
+            # link to PDF files. Skip them
+            # Examples:
+            # 2021 CO 55 No. 21SA125, In re Title, Ballot Title & Submission Clause for 2021-2022 #16
+            # https://www.cobar.org/For-Members/Opinions-Rules-Statutes/Colorado-Supreme-Court-Opinions/View/ArticleId/2374/2021-CO-55-No-21SA125-In-re-Title-Ballot-Title-Submission-Clause-for-2021-2022-16
+            # 2021 CO 60 No.21SA3, In re People v. Sprinkle
+            # https://www.cobar.org/For-Members/Opinions-Rules-Statutes/Colorado-Supreme-Court-Opinions/View/ArticleId/2383/2021-CO-60-No-21SA3-In-re-People-v-Sprinkle
```

Comment:
These could've been avoided by checking if they were PDFs?

Reply:
Lines **187 to 189** validate these cases:
https://github.com/freelawproject/juriscraper/blob/9de5eeecb36b516d7ce0be5691d3720a36e33b0b/juriscraper/opinions/united_states/state/colo.py#L187

Links:
> 2021 CO 55 No. 21SA125, In re Title, Ballot Title & Submission Clause for 2021-2022 #16
> https://www.cobar.org/For-Members/Opinions-Rules-Statutes/Colorado-Supreme-Court-Opinions/View/ArticleId/2374/2021-CO-55-No-21SA125-In-re-Title-Ballot-Title-Submission-Clause-for-2021-2022-16
> 2021 CO 60 No.21SA3, In re People v. Sprinkle
> https://www.cobar.org/For-Members/Opinions-Rules-Statutes/Colorado-Supreme-Court-Opinions/View/ArticleId/2383/2021-CO-60-No-21SA3-In-re-People-v-Sprinkle
> 

are subpages inside subpages. Those types of scenarios didn't existed last time the scraper was modifed.
 Also, when method `extract_pdf_url_and_name_from_resource` doesn't find a valid link to a PDF file, it returns false:
 https://github.com/freelawproject/juriscraper/blob/9de5eeecb36b516d7ce0be5691d3720a36e33b0b/juriscraper/opinions/united_states/state/colo.py#L176
I left the subpage `url` and `name` assignment (lines **173 to 178**) because it's working for previous scenarios.